trio_asyncio/_base.py

```diff
@@ -39,28 +39,6 @@ def clear(self):
         pass
 
 
-def _h_raise(handle, exc):
-    """
-    Convince a handle to raise an error.
-
-    trio-asyncio enhanced handles have a method to do this
-    but asyncio's native handles don't. Thus we need to fudge things.
```

Comment:
This was only used for add_reader/add_writer handles, which we create, so we know they're always ScopedHandles.